app/Modules/ExpertPanel/Models/ExpertPanel.php

```diff
@@ -199,37 +196,6 @@ public function evidenceSummaries()
         return $this->hasMany(EvidenceSummary::class);
     }
 
-    /**
-     * @return \Illuminate\Database\Eloquent\Relations\HasOne
-     */
-    public function leadershipAttestation()
-    {
-        return $this->hasOne(\App\Modules\ExpertPanels\Models\LeadershipAttestation::class);
-    }
-
-    /**
-     * @return \Illuminate\Database\Eloquent\Relations\HasOne
-     */
-    public function biocuratorOnboardingAttestation()
-    {
-        return $this->hasOne(\App\Modules\ExpertPanels\Models\BiocuratorOnboardingAttestation::class);
-    }
-
-    /**
-     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
-     */
-    public function curtionReviewProtocol()
-    {
-        return $this->belongsTo(\App\Modules\ExpertPanels\Models\CurtionReviewProtocol::class);
-    }
-
-    /**
-     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
-     */
```

Comment:
This never should have worked as is not a `group_id` attribute on the `ExpertPanel`.